extensions/identity-hub-verifier/src/main/java/org/eclipse/dataspaceconnector/identityhub/verifier/CredentialsVerifierExtension.java

```diff
@@ -21,16 +21,15 @@
 import org.eclipse.dataspaceconnector.identityhub.credentials.VerifiableCredentialsJwtServiceImpl;
```

Comment:
@marcgs @algattik 
Finally managed to make everything compile by refactoring the current extension in a more standard way. Creating @Provider methods depending on injected fields should be avoided in my opinion. 

@paullatzelsperger WDYT?